common/config/azure-pipelines/jobs/fast-ci.yaml

```diff
@@ -24,12 +24,42 @@ pr:
       - .github/CODEOWNERS
 
 jobs:
-  - template: ci-core.yaml
+- job:
+  variables:
+  - name: linuxPool
+    value: 'iModeljs Linux'
+  - name: mac_pool
+    value: iModelTechCI
+  - name: win_pool
+    value: iModelTechCI
+
+  strategy:
+    matrix:
+      "Windows_Node_16":
+        OS: windows-latest
+        platform: Windows_NT
+        name: $(win_pool)
+      "Linux_Node_16":
+        OS: ubuntu-latest
+        platform: Linux
+        name: $(linux_pool)
+      "MacOS_Node_16":
+        OS: macOS-latest
+        platform: Darwin
+        name: $(mac_pool)
+
+  pool:
+    name: $(name)
+    demands:
+      - Agent.OS -equals $(platform)
+
+  timeoutInMinutes: 120
```

Comment:
Can you change this to 60 minutes? @aruniverse and I discussed this in a separate issue.

Reply:
https://github.com/iTwin/itwinjs-core/pull/4286